interpreter/language/functions.go

```diff
@@ -60,6 +60,11 @@ var (
 			Value:     ifNotExists,
 			ForUpdate: true,
 		},
+		"list_append": &Function{
```

Comment:
can you add a test case for this function?

Reply:
added